app/views/associated_users/_user_form.html.haml

```diff
@@ -33,7 +33,7 @@
               = identity.display_name
             %a.btn.btn-xs.btn-danger.glyphicon.glyphicon-remove#new-associated-user-button{ href: "javascript:void(0);", data: { protocol_id: protocol.id, permission: 'true', toggle: "tooltip", animation: 'false', placement: 'right' }, title: "Remove" }
       %div#protocol_role_data{ data: { pr_id: identity.id, pr_name: identity.full_name, pi_id: current_pi.id, pi_name: current_pi.full_name }}
-      = form_for [:dashboard, project_role], url: project_role.new_record? ? dashboard_associated_users_path : dashboard_associated_user_path(project_role), remote: true, html: { class: "protocol_role_form" } do |form|
+      = form_for [:dashboard, project_role], url: project_role.new_record? ? associated_users_path : associated_user_path(project_role), remote: true, html: { class: "protocol_role_form" } do |form|
```

Comment:
We should try to keep the forms as a single file because the only difference between them is this `form_for` block. In both controllers (AssociatedUsersController && Dashboard::AssociatedUsersController) we could assign variables for the `for` and `url` params, then pass them to the form in the new.js/edit.js files.

For example
app/views/associated_users/_user_form.html.haml
```
= form_for entity, url: path, remote: true, html: { class: 'protocol_role_form' } do |form|
```

app/controllers/associated_users_controller
```
def new
    ...
    @entity = @project_role
    @path = associated_users_path
    ...
end

def edit
    ...
    @entity = @project_role
    @path = associated_user_path
    ...
```

app/controllers/dashboard/associated_users_controller
```
def new
    ...
    @entity = [:dashboard, @project_role]
    @path = dashboard_associated_users_path
   ...
end

def edit
    ...
    @entity = [:dashboard, @project_role]
    @path = dashboard_associated_user_path
     ...
end
```